eu.scarpetta.PDFMixTool.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: eu.scarpetta.PDFMixTool
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+command: pdfmixtool
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --device=dri
+- --filesystem=home:create
```

Comment:
`:create` makes no sense in this context; have you checked if it works properly with desktop portal so it doesn't need legacy permissions like this?